### DISCUSSION TITLE
unable to sign up/in - db error

### DISCUSSION THREAD

**Discussion by ibigbug** [mastodon/mastodon#25673] [Q&A]
I'm seeing below db errors in log

```
[38e81433-8025-4797-9c99-1c449903bd5c] ActiveRecord::StatementInvalid (PG::UndefinedColumn: ERROR:  column accounts.silenced does not exist
LINE 1: ...ader_updated_at", "accounts"."avatar_remote_url", "accounts"...
                                                             ^
HINT:  Perhaps you meant to reference the column "accounts.silenced_at".
):
```

I ran 

```
RAILS_ENV=production bundle exec rails db:migrate
```

and there is no output

now when I try to sign in to an account that I just created via cli, it goes to an error page.

any ideas?

**Comment by vmptk** [comment]
I am facing the same issue... All the db migration seems to be applied. However, as I am using Docker image I have noticed that the db/migrate/* files are not the complete list (when the tag is done), e.g. the last migration applied was from 2022-12...
